code/game/gamemodes/game_mode.dm

```diff
@@ -459,7 +459,7 @@ var/global/list/additional_antag_types = list()
 		else if(man.mind.assigned_job)
 			var/datum/job/job = man.mind.assigned_job
 			var/evil = 0
-			if(man.client.prefs.nanotrasen_relation == COMPANY_OPPOSED || man.client.prefs.nanotrasen_relation == COMPANY_SKEPTICAL)
+			if(man.client.prefs.economic_status == ECONOMIC_POOR || man.client.prefs.economic_status == ECONOMIC_UNDERPAID)
```

Comment:
The things here don't really make sense if it's economy status, tbh. I'd axe this proc.